lib/websocket/websocket-handler.ts

```diff
@@ -152,7 +158,7 @@ function setup(socket: Interface.ISocket): void
                         activeSubscriptions.set(s.correlationId, s);
                     });
                     socket.log.debug('Subscribed');
-                    socket.notifySubscribed();
+                    socket.notifySubscribed(getCorrelationIds(subscriptions));
```

Comment:
I like the separate loop because it is clear on the intent of this function and is not conflated with other logic.  I doubt we will have a high-rate of `subscribe` calls where this would show up as a performance concern.